src/libraries/System.Numerics.Vectors/ref/System.Numerics.Vectors.cs

```diff
@@ -596,6 +597,7 @@ public partial struct Vector4 : System.IEquatable<System.Numerics.Vector4>, Syst
         public Vector(T[] values, int index) { throw null; }
         public static System.Numerics.Vector<T> AllBitsSet { get { throw null; } }
         public static int Count { get { throw null; } }
+        public static System.Numerics.Vector<T> Indices { get { throw null; } }
```

Comment:
There are several places that this should be usable including in LINQ and in Tensors

However, I was going to put up an issue and cover them in a follow up PR since source.dot.net wasn't bringing them up at the moment (making finding all the places rather difficult): https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs,2157880a86aea82e,references